build-in-container/scripts/build-mariner.sh

```diff
@@ -6,10 +6,31 @@
 set -e
 set -x
 
+# parse args passed to container
+while (( "$#" )); do
+    case "$1" in
+        --container_type)
+        if [ -n "$2" ] && [ ${2:0:1} != "-" ]; then
```

Comment:
Having a comment here explaining what this this is meant to match would be very helpful.

Reply:
Did you mean something like this?
# arguments:
#
# --container_type                        : container type is either 'build' or 'interactive'
# --RPM_repo_file                         : path to custom repo file
# --RPM_container_URL                : comma delimited URLs to Azure containers with RPMs
# --enable_custom_repofile          : enable installing RPMs from repo(s) listed in RPM_repo_file, if true
# --enable_custom_repo_storage : enable installing RPMs from container(s) in RPM_container_URL, if true
# --disable_mariner_repo             : disable installing RPMs from default Mariner PMC, if true
#